PhotoPoints/app/build.gradle

```diff
@@ -29,6 +29,7 @@ dependencies {
     implementation 'androidx.navigation:navigation-fragment:2.1.0'
```

Comment:
I get error: "Canvas: trying to draw too large(209285016bytes) bitmap." when trying to start the app using a Google Pixel 2 emulator. 

You might want to add the image in the right folder (xxhdpi etc...) so that the framework can pick the right format for it using the R.mipmap instead of R.drawable. 

I just tested this and that should do the trick.

Reply:
Moved to mipmap